commit.go

```diff
@@ -181,7 +181,9 @@ func Push(ctx context.Context, image string, dest types.ImageReference, options
 		return errors.Wrapf(err, "error copying layers and metadata")
 	}
 	if options.ReportWriter != nil {
-		fmt.Fprintf(options.ReportWriter, "\n")
+		fmt.Fprintf(options.ReportWriter, "")
 	}
+	digest := "@" + img.Digest.Hex()
+	fmt.Printf("Succefully pushed %s%s\n", dest.StringWithinTransport(), digest)
```

Comment:
Succefully to Successfully

Reply:
Updated. And the running result is added in the commit information, I hope this is the result you want. Please tell me if there is any problem, and I will change it again.